draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -6897,72 +6897,94 @@ Accept-Language: da, en-gb;q=0.8, en;q=0.7
     </t>
   </li>
   <li>
     <t>
        To inform user agent recipients that this response is subject to
-       content negotiation (<xref target="content.negotiation"/>) and that a
+       content negotiation (<xref target="content.negotiation"/>) and a
        different representation might be sent in a subsequent request if
        additional parameters are provided in the listed header fields
        (<x:ref>proactive negotiation</x:ref>).
     </t>
   </li>
 </ol>
 <t>
-   An origin server &SHOULD; send a Vary header field when its algorithm for
-   selecting a representation varies based on aspects of the request message
-   other than the method and target URI, unless the variance cannot be
-   crossed or the origin server has been deliberately configured to prevent
-   cache transparency. For example, there is no need to send the Authorization
-   field name in Vary because reuse across users is constrained by the field
-   definition (<xref target="field.authorization"/>). Likewise, an origin server might use
-   Cache-Control response directives (<xref target="field.cache-control"/>) to supplant Vary if it
-   considers the variance less significant than the performance cost of Vary's
-   impact on caching.
+   An origin server &SHOULD; send a Vary header field when doing so might
+   improve the recipient's content selection in a later request, unless
+   the resource has been deliberately configured to prefer cache reuse
+   regardless of changes to content negotiation preferences.
+</t>
+<t>
+   Vary is intended to prevent (until validated by the origin server)
+   a cached response from being reused to satisfy a subsequent request
+   that has different preferences in the selecting header fields.
+   However, this is a choice made by the origin server and communicated to
+   recipients, not an interoperability requirement whenever content
+   selection is present. Vary might be elided when an origin server
+   considers the variance in content selection to be less significant than
+   Vary's performance impact on caching, particularly when such variance
+   is already being limited by Cache-Control response directives
+   (<xref target="field.cache-control"/>).
+</t>
+<t>
+   Cached content selection is only improved by Vary if there is some
+   chance that the origin server's selection algorithm might prefer a
+   different representation for the recipient in a subsequent request.
+   For example, there is no need to send the Authorization field name in
+   Vary because reuse of that response for a different user is prohibited
+   by the field definition (<xref target="field.authorization"/>).
+   Likewise, if the response content has been selected by network region
+   but the origin server wants the cached response to be reused even if
+   recipients move from one region to another, then there is no need for
+   the origin server to indicate such variance in Vary.
```

Comment:
```suggestion
   There is no need to send the Authorization field name in
   Vary because reuse of that response for a different user is prohibited
   by the field definition (<xref target="field.authorization"/>).
   Likewise, if the response content has been selected or influenced by
   by network region but the origin server wants the cached response to 
   be reused even if recipients move from one region to another, then 
   there is no need for the origin server to indicate such variance in Vary.
```

Reply:
So, you are advocating removal of all the explanations and the parts that are not cache-specific and the forward references to Caching? How is this addressing the issue raised?